src/client/search/SearchActions.js

```diff
@@ -22,3 +25,24 @@ const addQueryParamTypes = (searchQuery, query, paramTypes) => {
     paramTypes,
   };
 };
+
+export const INVALIDATE_FEED_RESULTS = 'INVALIDATE_FEED_RESULTS';
+export const invalidateFeedResults = () => {
+  return (dispatch, getState) => {
+    dispatch({ 'type': INVALIDATE_FEED_RESULTS });
+    dispatch(searchApiForFeed(getState().searchTerms));
+  };
+};
+
+export const UPDATE_FEED_RESULTS = 'UPDATE_FEED_RESULTS';
```

Comment:
Ohh I see, this is when you actually receive the results. Was confused because of the searchApi for Feed method below was promising to fetch the post, would prefer we use RECIEVE to update then here.

Then I'm confused what invalidateFeedResults is actually doing right now or is planning to do ? you're dispatching an action for it in line 32, but not actually doing anything in the reducer with it ? :confused: